builds/fabric8-che/assembly/assembly-wsmaster-war/pom.xml

```diff
@@ -22,14 +22,14 @@
     </properties>
     <dependencies>
         <dependency>
-            <groupId>aopalliance</groupId>
-            <artifactId>aopalliance</artifactId>
-        </dependency>
-        <dependency>
             <groupId>com.google.inject</groupId>
             <artifactId>guice</artifactId>
         </dependency>
         <dependency>
+            <groupId>aopalliance</groupId>
```

Comment:
This is related to a pom sorting error I encountered during the work on this PR.